Blockzilla/Assets.xcassets/icon_private_mode.imageset/Contents.json

```diff
@@ -11,7 +11,7 @@
           "value" : "dark"
         }
       ],
-      "filename" : "Private mode-24.pdf",
+      "filename" : "Private Mode-24.pdf",
```

Comment:
I noticed this asset name changing between PRs. Does it fix/break something?

Reply:
I tried to solve an issue regarding assets. Since I started using the latest xcode this image was not placed properly in their slot for dark mode and it did not appeared on onboarding screen for dark mode, so I added it again. I hope it solves the issue.